thirdparty/imgui_suite/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+
+if(NOT glfw3_POPULATED)
+#     fetchcontent_populate(glfw3)
+    set(SAVE_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS})
+    set(BUILD_SHARED_LIBS OFF)
+    set(GLFW_INSTALL OFF)
+    add_subdirectory("glfw")
+    set_property(TARGET glfw PROPERTY POSITION_INDEPENDENT_CODE ON)
+    set(BUILD_SHARED_LIBS ${SAVE_BUILD_SHARED_LIBS})
+    export(TARGETS glfw FILE glfw.cmake NAMESPACE glfw::)
+endif()
+
+# Add imgui directly to our build.
+file(GLOB imgui_sources imgui/cpp/*.cpp imgui/cpp/misc/cpp/*.cpp)
+file(GLOB implot_sources implot/cpp/*.cpp)
+file(GLOB fonts_sources generated/fonts/src/*.cpp)
+set(imgui_all_sources
+    ${imgui_sources}
+    ${implot_sources}
+    ${fonts_sources}
+    imgui/cpp/backends/imgui_impl_glfw.cpp
+    imgui/cpp/backends/imgui_impl_opengl3.cpp
+    generated/gl3w/src/gl3w.c
+    stb/cpp/stb_image.cpp
+)
+if(MSVC)
+    add_library(imgui STATIC ${imgui_all_sources})
+else()
+    add_library(imgui ${imgui_all_sources})
+endif()
+target_compile_definitions(imgui PUBLIC IMGUI_IMPL_OPENGL_LOADER_GL3W)
+if(MSVC)
+    target_sources(imgui PRIVATE imgui/cpp/backends/imgui_impl_dx11.cpp)
+else()
+    if(APPLE)
+        target_compile_options(imgui PRIVATE -fobjc-arc)
+        set_target_properties(imgui PROPERTIES LINK_FLAGS "-framework Metal -framework QuartzCore")
```

Comment:
Include the `LinkMacOSGUI` macro and use `wpilib_link_macos_gui`.

Reply:
The `wpilib_link_macos_gui` macro replaces `set_target_properties(imgui PROPERTIES LINK_FLAGS "-framework Metal -framework QuartzCore")`, so that line can just be removed.